.github/workflows/examples.yml

```diff
@@ -101,6 +101,8 @@ jobs:
       - name: Install package
         shell: bash -l {0}
         run: |
+          pip install git+https://github.com/openforcefield/openff-interchange.git@skip-drivers
+
```

Comment:
To be explicit: I will include this branch into a release before aiming to have this PR merged

Reply:
This is now included in v0.1.3; CI will need to be restarted next week to get it to pass (the package will not be online until after hours)